.github/workflows/tfpackage.yml

```diff
@@ -9,8 +9,10 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-        with: { fetch-depth: 0 }
+        uses: actions/checkout@v3
+
+      - name: Unshallow
+        run: git fetch --prune --unshallow
```

Comment:
I think goreleaser is sensitive to having all the tags available. it uses it to calculate semver I think

Reply:
Correct, and that gets you all the tags.